### PR TITLE
add configKey and userConfig page; support download all result set wh…

### PR DESCRIPTION
**Pull request by mayinrain**
…ose type is csv; something optimization; bug fix

### What is the purpose of the change

1. add new pages(config Key, userConfig)
2. support download all result set(csv)
3. i18n optimization and ui optimization
4. fix bug(reading property of undifined when not exsit engine list)

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [Linkis mailing list](https://linkis.apache.org/community/how-to-subscribe) first)
- [ ] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

File: linkis-web/src/apps/linkis/assets/styles/console.scss [hunk 1]
```diff
@@ -17,8 +17,29 @@
...
+//  * {
 
+//     font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "PingFang SC", "Hiragino Sans GB", "Microsoft YaHei", "Helvetica Neue", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "JinbiaoSong", "JinbiaoSongExt";
+//   }
+body {
+    font-family: "Helvetica Neue",Helvetica,"PingFang SC","Hiragino Sans GB","Microsoft YaHei","微软雅黑",Arial,sans-serif, "JinbiaoSong", "JinbiaoSongExt";
+}
 .console-page{
+    .ivu-input {
+        font-family: Arial, -apple-system, BlinkMacSystemFont, "Segoe UI", "PingFang SC", "Hiragino Sans GB", "Microsoft YaHei", "Helvetica Neue", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "JinbiaoSong", "JinbiaoSongExt";
```
Comment (lucaszhu2zgf) [inline linkis-web/src/apps/linkis/assets/styles/console.scss:30]: why use another font 'Arial'

File: linkis-web/src/apps/linkis/module/acrossClusterRule/index.vue [hunk 2]
```diff
@@ -0,0 +1,543 @@
...
+        let params = {};
+        const keys = Object.keys(this.confirmQuery);
+        for (let i = 0; i< keys.length; i++) {
+          if(this.confirmQuery[keys[i]]) {
+            params[[keys[i]]] = this.confirmQuery[keys[i]];
+          }
+        }
+        params.pageNow = this.page.pageNow;
+        params.pageSize = this.page.pageSize;
+        await api.fetch("/configuration/acrossClusterRule/list", params, "get")
```
Comment (lucaszhu2zgf) [inline linkis-web/src/apps/linkis/module/acrossClusterRule/index.vue:349]: use format `const res = await api.fetch()` as it was inside a  async function

File: linkis-web/src/apps/linkis/module/acrossClusterRule/index.vue [hunk 3]
```diff
@@ -0,0 +1,543 @@
...
+      } catch(err) {
+        window.console.log(err);
+        this.tableLoading = false;
+      }
+
+    },
+    async init() {
+      this.loading = true;
+      await this.getTableData();
+      this.loading = false;
```
Comment (lucaszhu2zgf) [inline linkis-web/src/apps/linkis/module/acrossClusterRule/index.vue:369]: check if `this.loading = false;` will excute  when this.getTableData excute error

File: linkis-web/src/apps/linkis/module/configManagement/index.vue [hunk 4]
```diff
@@ -0,0 +1,532 @@
...
+        if(this.confirmQuery?.engineType === 'all') delete this.confirmQuery.engineType;
+        const keys = Object.keys(this.confirmQuery);
+        for (let i = 0; i< keys.length; i++) {
+          if(this.confirmQuery[keys[i]]) {
+            params[[keys[i]]] = this.confirmQuery[keys[i]];
+          }
+        }
+        params.pageNow = this.page.pageNow;
+        params.pageSize = this.page.pageSize;
+        await api.fetch("/configuration/baseKeyValue", params, "get")
```
Comment (lucaszhu2zgf) [inline linkis-web/src/apps/linkis/module/configManagement/index.vue:350]: use format `const res = await api.fetch()` as it was inside a async function

File: linkis-web/src/apps/linkis/module/userConfig/index.vue [hunk 5]
```diff
@@ -0,0 +1,523 @@
...
+        let params = {};
+        const keys = Object.keys(this.confirmQuery);
+        for (let i = 0; i< keys.length; i++) {
+          if(this.confirmQuery[keys[i]]) {
+            params[[keys[i]]] = this.confirmQuery[keys[i]];
+          }
+        }
+        params.pageNow = this.page.pageNow;
+        params.pageSize = this.page.pageSize;
+        await api.fetch("/configuration/userKeyValue", params, "get")
```
Comment (lucaszhu2zgf) [inline linkis-web/src/apps/linkis/module/userConfig/index.vue:308]: use format `const res = await api.fetch()` as it was inside a async function

File: linkis-web/src/apps/linkis/assets/styles/console.scss (hunk 1 above)
Comment (mayinrain) [reply linkis-web/src/apps/linkis/assets/styles/console.scss:30]: It's style of iview input component, I just supplement jinbiaosong and jinbiaosongext font

**Pull request merged**